lib/tasks/docs.thor

```diff
@@ -341,7 +337,7 @@ class DocsCLI < Thor
 
   def download_doc(doc)
     target_path = File.join(Docs.store_path, doc.path)
-    open "http://dl.devdocs.io/#{doc.path}.tar.gz" do |file|
+    open "https://downloads.devdocs.io/#{doc.path}.tar.gz" do |file|
```

Comment:
Interestingly, some downloads fail (the failing docs got an update on 2021-01-30):

```sh
$ bundle exec thor docs:download vue
(1/1) Vue.js 3 FAILED (OpenURI::HTTPError: 403 Forbidden)
Done

$ bundle exec thor docs:download vuex
(1/1) Vuex 4 FAILED (OpenURI::HTTPError: 403 Forbidden)
Done

$ bundle exec thor docs:download erlang
(1/1) Erlang 23 FAILED (OpenURI::HTTPError: 403 Forbidden)
Done

$ bundle exec thor docs:download python
(1/1) Python 3.9 OK
Done
```

Reply:
Ah, yes, good point.  The final thing I wanted to do was to sync the old bucket (which you have been accessing) with the new ones I set up for the migration.  I'll let you know when this is good to go.